src/commonTest/kotlin/exchange.dydx.abacus/payload/v4/V4TradeInputTests.kt

```diff
@@ -1174,4 +1175,804 @@ open class V4TradeInputTests : V4BaseTests() {
         )
 
     }
+
+
+    private fun testExecution() {
+        testExecutionStopLimit()
```

Comment:
Are there dependencies between those tests?  If not, can we make them public instead of private, so we can run them separately?